src/components/ColorPicker.js

```diff
@@ -72,5 +72,3 @@ class ColorPicker extends React.Component {
         )
     }
 }
-
-export default ColorPicker;
```

Comment:
Again, just for me to understand, any reason not to use the default export?

Reply:
Preferring named exports for module control. default exports are actually okay, but with multiple exports from same module, it forces the import syntax like `import Module, { OtherExports } from 'module'`. With named exports, we always do `import { Module, OtherExports } from 'module'`

Reply:
Ok, thanks for the explanation!